lib/openapi_parser/schemas/openapi.rb

```diff
@@ -29,9 +29,13 @@ def initialize(raw_schema, config, uri: nil, schema_registry: {})
     #   @return [Components, nil]
     openapi_attr_object :components, Components, reference: false
 
+    # @!attribute [r] servers
+    #   @return [Servers, nil]
+    openapi_attr_list_object :servers, Servers, reference: false
+
     # @return [OpenAPIParser::RequestOperation, nil]
     def request_operation(http_method, request_path)
-      OpenAPIParser::RequestOperation.create(http_method, request_path, @path_item_finder, @config)
+      OpenAPIParser::RequestOperation.create(http_method, request_path, @path_item_finder, @config, servers)
```

Comment:
I think we don't need to pass Server Object.

The RequestOperation object for validating using Operation Object.
The Operation Object specified by path and http method so it don't care about server URL so they don't need to Server Object which define server URL.
(This object is intended to run on a server that accepts requests. This object is supposed to run on the server that accepts requests, although it can also be used on the client side.)

I think we should add validate method to Server Object and call it from OpenAPI Object.
(The client validate URL using that method before send request )